distribution/docs/src/main/resources/content/_configuring/hiding-errors-by-role.adoc

```diff
@@ -9,13 +9,14 @@
 
 Prevent certain users from seeing data with certain types of errors or warnings.
 Typically, this is used for security markings.
-Metacards with the specified metacard attributes will be hidden from users without the specified user attributes.
+If the *Metacard Validation Filter Plugin* is configured to *Filter errors* and/or *Filter warnings*, metacards with errors/warnings will be hidden from users without the specified user attributes.
 
 . Navigate to the *${admin-console}*.
 . Select the *${ddf-catalog}* application.
 . Select *Configuration*.
 . Select *Metacard Validation Filter Plugin*.
 . For *Attribute map*, enter both the metacard `SECURITY` attribute to filter and the user attribute to filter.
-.. `<SECURITY ATTRIBUTE>=<USER ATTRIBUTE>`.
-. Select *Filter errors* to filter errors. Users without the security attribute will not see metacards with errors.
-. Select *Filter warnings* to filter warnings. Users without the security attribute will not see metacards with warnings.
+.. `invalid-state=<USER ROLE>`.
```

Comment:
I'm hesitant to hardcode "invalid-state" since the security attribute used here can be changed. Trying to locate how to configure that

Reply:
Looks like the this default value is being set in this config file https://github.com/codice/ddf/blob/850db96bd3bca1b75bd94f9a5c949bed20c1cd71/platform/security/pdp/security-pdp-authzrealm/src/main/resources/default-authzrealm.config
So dropping in a new one or altering this one would change it and make the docs inaccurate. Maybe we should mention this configuration?

Reply:
I can revert to the generic version and add a configuration step for setting the attribute.. before this one? after?

Reply:
I think I like that. At least keeping it generic. If we already have docs on setting that config then all we'd probably need to do is mention it should map to whatever security attribute is mapped to "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role"... except that can be changed too. In fact, it probably doesn't have to be roles at all. We could configure this to only allow based on email if we wanted...

Reply:
Maybe that's why it was just generic `SECURITY ATTRIBUTE` to begin with. I wish we could be more clear but it may just serve to confuse users

Reply:
So, I think we're opting at this time to just call "invalid-state" the "default" value and leave out updating the configuration file until we have a use case for updating it. I'm fine with encouraging people to use that default value in most, if not all, cases.